pom.xml

```diff
@@ -93,6 +93,14 @@
             <name>ForgeRock Private Release Repository</name>
             <url>https://maven.forgerock.org/artifactory/private-releases</url>
         </repository>
+        <repository>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+            <id>forgerock-internal-snapshots</id>
+            <name>ForgeRock Internal Snapshots Repository</name>
+            <url>https://maven.forgerock.org/artifactory/internal-snapshots</url>
```

Comment:
Without this change, this repository is referencing AM 7.5.0, which doesn't align with P1AIC releases.